.circleci/config.yml

```diff
@@ -0,0 +1,71 @@
+version: 2.1
+
+executors:
+  mac-executor:
+    macos:
+      xcode: "11.2.1"
+    working_directory: ~/code
+
+jobs:
+  install_dependendies:
+    executor: mac-executor
+    steps:
+      - checkout
+      - restore_cache:
+          keys:
+            - gem-cache-v1-{{ checksum "Gemfile.lock" }}
+            - gem-cache-v1
+      - run:
+          name: Bundler
+          command: |
+            bundle install --path vendor/bundle --jobs 4 --retry 3
+      - save_cache:
+          key: gem-cache-v1-{{ checksum "Gemfile.lock" }}
+          paths:
+            - vendor/bundle
+      - persist_to_workspace:
+          root: .
+          paths:
+            - .
+  test:
+    executor: mac-executor
+    steps:
+      - checkout
+      - attach_workspace:
+          at: ~/code
+      - run:
+          name: Test
+          command: |
+            bundle config --local path vendor/bundle
+            bundle exec fastlane test
+      - run:
+          name: Sonarqube
+          command: |
+            bundle config --local path vendor/bundle
+            bundle exec fastlane sonarqube
+      - store_test_results:
+          path: fastlane/test_output
```

Comment:
I couldn't find the test summary on CircleCI, maybe we need to rename the ` report.junit` to `report.xml` for it to work? According to the docs, it should work like this, only difference is that [their files have ` .xml` as file endings](https://circleci.com/docs/2.0/configuration-reference/#store_test_results).

Reply:
That is because all tests passed. Failing tests would show up here as well. fastlane produces JUnit XML compatible output all good. Documentation can be found [here](https://circleci.com/docs/2.0/collect-test-data/).

![Screen Shot 2020-02-17 at 10 11 50](https://user-images.githubusercontent.com/24912807/74639289-f57b3900-516d-11ea-8350-33ef25524465.png)